lib/placebo/meck.ex

```diff
@@ -98,7 +98,7 @@ defmodule Placebo.Meck do
       |> Process.info(:dictionary)
       |> get_in([Access.elem(1), :"$callers", default_value([])])
 
-    [pid | callers]
+    [pid] ++ [callers] |> List.flatten()
```

Comment:
I'm not 100% sure about this but I think the better solution is to change the `default_value/1` function so the second pattern match returns a list.

```elixir
:get, data, next ->
  next.([data])
```

And then this line can change to `List.flatten([pid | callers])`.